core/src/main/resources/error/error-classes.json

```diff
@@ -851,6 +851,11 @@
       "Cannot name the managed table as <identifier>, as its associated location <location> already exists. Please pick a different table name, or remove the existing location first."
     ]
   },
+  "MALFORMED_CSV_RECORD" : {
+    "message" : [
+      "Malformed CSV record. The number of tokens <tokenLength> doesn't match the schema <schemaLength>."
```

Comment:
Can we output the malformed CSV record? If an user process billions of CSV lines, how he/she should figure out which one is bad?

Reply:
Seems like this exception is triggered when checking the schema before parsing the actual input row.

Maybe should we fix the error class name and message more properly?

For example:
```json
  "MALFORMED_CSV_SCHEMA" : {
    "message" : [
      "Malformed CSV schema. The number of tokens <tokenLength> doesn't match the schema <schemaLength>."
    ]
  }
```

Or maybe we can show the malformed schema itself:
```json
  "MALFORMED_CSV_SCHEMA" : {
    "message" : [
      "Malformed CSV schema. <malformedSchema>."
    ]
  }
```

This will display look like:
```
[MALFORMED_CSV_RECORD] Malformed CSV schema: StructType(StructField(year,StringType,true),StructField(make,StringType,true),StructField(model,StringType,true),StructField(comment,StringType,true),StructField(blank,StringType,true))
```

Reply:
This one process actual CSV records:
```
  private def convert(tokens: Array[String]): Option[InternalRow] = {
    if (tokens == null) {
      throw BadRecordException(
        () => getCurrentInput,
        () => None,
        QueryExecutionErrors.malformedCSVRecordError())
    }
```
and `getCurrentInput` can get you a bad record. Can't it?

Reply:
Oh... I didn't know `getCurrentInput`. Thanks!